modules/_labs/color-picker/react/lib/ColorPicker.tsx

```diff
@@ -117,6 +122,7 @@ const isCustomColor = (colors: string[], hexCode?: string) => {
 const ColorPicker = ({
   colorSet = defaultColors,
   customHexInputLabel = 'Custom Hex Color',
+  submitLabel = 'Submit',
```

Comment:
I'm not sure why this label was shared with the input. The label didn't describe the functionality any different than the input. This also made it harder to test.